utils/src/unit-test/java/uk/gov/moj/sdt/utils/web/filter/ContextCleanupFilterTest.java

```diff
@@ -88,33 +99,24 @@ public void setUp() {
      * @throws IOException      the exception thrown on IO problems
      */
     @Test
-    public void doFilterTest() throws IOException, ServletException {
+    void doFilterTest() throws IOException, ServletException {
         // Add code to set something in SdtContext.
         SdtContext.getContext().setRawInXml("TestXml");
 
-        Assert.assertNotNull(SdtContext.getContext().getRawInXml());
+        assertNotNull(SdtContext.getContext().getRawInXml());
 
         contextCleanupFilter = new ContextCleanupFilter();
 
         mockFilterChain.doFilter(mockServletRequest, mockServletResponse);
 
-        EasyMock.expectLastCall();
-
-        EasyMock.replay(mockServletRequest);
-        EasyMock.replay(mockServletResponse);
-        EasyMock.replay(mockFilterChain);
-
         contextCleanupFilter.doFilter(mockServletRequest, mockServletResponse, mockFilterChain);
 
-        EasyMock.verify(mockServletRequest);
-        EasyMock.verify(mockServletResponse);
-        EasyMock.verify(mockFilterChain);
+        verify(mockFilterChain, times(2)).doFilter(mockServletRequest, mockServletResponse);

```

Comment:
Does this also verify the mockServletRequest and mockServletResponse objects as the previous version did?

Reply:
That's how I read it. verify mockObject on its own no longer exists so looked for a solution.